spec/rails_helper.rb

```diff
@@ -63,7 +63,3 @@
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
 end
-
-def last_updated_dataset
```

Comment:
It would be preferable to propose changes like this in a separate pull-request.

Reply:
I agree - I intended to do it as part of my let! PR but forgot it. I will try to remember better in future.